packages/odie-client/src/components/message/messages-container.tsx

```diff
@@ -89,7 +89,7 @@ export const MessagesContainer = ( { currentUser }: ChatMessagesProps ) => {
 		}
 
 		( chat?.status === 'loaded' || chat?.status === 'closed' ) && setChatMessagesLoaded( true );
-	}, [ chat, isForwardingToZendesk, hasForwardedToZendesk ] );
+	}, [ chat.status, isForwardingToZendesk, hasForwardedToZendesk ] );
```

Comment:
Can't this throw? Considering line 91? And this can still run excessively. 

I'd do 

```js
const isChatLoadedOrClosed = chat?.status === 'loaded' || chat?.status === 'closed';
```
Then use that value in the effect. 

In general, my approach would be to probably delete `setChatMessagesLoaded` as a state and just make it a variable. 

```
const chatMessagesLoaded = isForwardingToZendesk || hasForwardedToZendesk ? false : chat?.status === 'loaded' || chat?.status === 'closed' 
```

Reply:
I totally agree but I'm reluctant to touch this part too much in this PR
Pushed a safety check for the chat.status